tests/test_fabricpath_global.rb

```diff
@@ -42,213 +40,203 @@ def no_feature_fabricpath
+    fg.aggregate_multicast_routes = true
+    assert(fg.aggregate_multicast_routes,
+           'aggregate_multicast_routes: Expected: true')
+
+    fg.aggregate_multicast_routes = false
+    refute(fg.aggregate_multicast_routes,
+           'aggregate_multicast_routes: Expected: false')
   end
 
   def test_allocate_delay
-    @global = FabricpathGlobal.new('default')
-    # test default value
-    assert_equal(@global.default_allocate_delay,
-                 @global.allocate_delay,
-                 'Default allocate_delay not set correctly')
-    @global.allocate_delay = 20
-    assert_equal(20, @global.allocate_delay,
-                 'allocate_delay not set to 20')
+    fg = FabricpathGlobal.new('default')
+    assert_equal(fg.default_allocate_delay, fg.allocate_delay)
+
+    fg.allocate_delay = 20
+    assert_equal(20, fg.allocate_delay)
   end
 
   def test_graceful_merge
-    @global = FabricpathGlobal.new('default')
-    # test default value
-    assert_equal(:enable, @global.graceful_merge,
-                 'Default graceful_merge not set correctly')
-    @global.graceful_merge = :disable
-    assert_equal(:disable, @global.graceful_merge,
-                 'graceful merge not set to disable')
+    fg = FabricpathGlobal.new('default')
+    assert_equal(fg.default_graceful_merge, fg.graceful_merge)
+
+    fg.graceful_merge = :disable
+    assert_equal(:disable, fg.graceful_merge)
   end
 
   def test_linkup_delay_all
-    @global = FabricpathGlobal.new('default')
-    # test default value
-    assert_equal(@global.default_linkup_delay,
-                 @global.linkup_delay,
-                 'Default linkup_delay not set correctly')
-    @global.linkup_delay = 25
-    assert_equal(25, @global.linkup_delay,
-                 'linkup_delay not set to 25')
-
-    return if n5k6k_platforms?
-
-    refute(@global.linkup_delay_always,
-           'linkup_delay_always should not be set by default')
-    @global.linkup_delay_always = true
-    assert(@global.linkup_delay_always,
-           'linkup_delay_always is not getting set')
-
-    @global.linkup_delay_enable = true
-    @global.linkup_delay_enable = false
-    refute(@global.linkup_delay_enable,
-           'linkup_delay is not getting disabled')
+    fg = FabricpathGlobal.new('default')
+    assert_equal(fg.default_linkup_delay, fg.linkup_delay)
+
+    fg.linkup_delay = 25
+    assert_equal(25, fg.linkup_delay)
+
+    if validate_property_excluded?('fabricpath', 'linkup_delay_always')
+      assert_raises(Cisco::UnsupportedError) { fg.linkup_delay_always = true }
+    else
+      assert_equal(fg.default_linkup_delay_always, fg.linkup_delay_always)
+      fg.linkup_delay_always = true
+      assert(fg.linkup_delay_always, 'linkup_delay_always: Expected: true')
+      fg.linkup_delay_always = false
+      refute(fg.linkup_delay_always, 'linkup_delay_always: Expected: false')
+    end
+
+    if validate_property_excluded?('fabricpath', 'linkup_delay_enable')
```

Comment:
I considered it and decided it was better to explicitly check for excluded on each property.